build.sbt

```diff
@@ -80,15 +59,23 @@ lazy val faDbCore = (project in file("core"))
     libraryDependencies ++= coreDependencies(scalaVersion.value),
     (Compile / compile) := ((Compile / compile) dependsOn printScalaVersion).value // printScalaVersion is run with compile
   )
-  .settings(jacocoCoreReportSettings: _*)
+  .settings(
+    jacocoReportSettings := commonJacocoReportSettings.withTitle("fa-db:core Jacoco Report"),
+    jacocoExcludes := commonJacocoExcludes ++ Seq(
+//      "za.co.absa.fadb.naming_conventions.AsIsNaming" // extra exclude example
```

Comment:
This one, I would leave out.
Examples were given above, how to do them. And extending common `val Seq` this way is standard approach in _sbt_

Reply:
Additional example of exclude removed in commit 722618e324218cb45457c35a47959a37cfaffd00.